src/styles/saas-list.less

```diff
@@ -1,53 +1,93 @@
+.build-applications-view {
+
+  h1 {
+    color: @saas-text;
+    margin: 0;
+    width: 100%;
+    padding: 20px;
+    padding-left: @grid-gutter-width;
+  }
+}
+
 .saas-offerings-container {
-  background: @color-pf-black-200;
-  padding-left: 0px;
 
   .spinner-container {
     padding-top: 60px;
     padding-bottom: 60px;
   }
 
   .card-view-pf {
-    padding-left: 21px;
-  }
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-around;
+    flex-direction: column;
+    padding: 0;
+    margin: 0 (@grid-gutter-width / 2);
 
-  .card-view-pf .card-content {
-    background-color: @color-pf-white;
-    cursor: pointer;
-    margin: 0;
+    .card-content {
+      cursor: pointer;
```

Comment:
Yep, my suggestion would be that the name on the tile be an `<a>` tag.  Then if we want to style it to not look like an `<a>` we can. I'm opening https://github.com/openshift/origin-web-catalog/issues/67 since this is an issue with more than just the SaaS tiles.